test/data/testinit-jsdom.js

```diff
@@ -1,4 +1,4 @@
-"use strict";
```

Comment:
Why remove strict mode from this file?

Reply:
Thanks for catching that. I ran `eslint --fix` at some point and some accidental changes got past me.